### PR TITLE
set image override for manual service run if provided

### PR DESCRIPTION
**Pull request by ianedwards**

### CODE REVIEW

File: api/server/handlers/porter_app/run_app_job.go [hunk 1]
```diff
@@ -86,11 +91,30 @@ func (c *RunAppJobHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 	telemetry.WithAttributes(span, telemetry.AttributeKV{Key: "deployment-target-id", Value: request.DeploymentTargetID})
 
+	var commandOptional *string
+	if request.RunCommand != "" {
+		commandOptional = pointer.String(request.RunCommand)
```
Comment (d-g-town) [inline api/server/handlers/porter_app/run_app_job.go:96]: nit: why not just use `&request.RunCommand`?

File: api/server/handlers/porter_app/run_app_job.go [hunk 2]
```diff
@@ -86,11 +91,30 @@ func (c *RunAppJobHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
...
+	}
+
+	var imageOverrideOptional *porterv1.AppImage
+	if request.Image.Repository != "" || request.Image.Tag != "" {
```
Comment (d-g-town) [inline api/server/handlers/porter_app/run_app_job.go:100]: personally don't like the idea of just changing repository and not tag

**Review by d-g-town — APPROVED** [review]

File: api/server/handlers/porter_app/run_app_job.go (hunk 2 above)
Comment (ianedwards) [reply api/server/handlers/porter_app/run_app_job.go:100]: fair

File: api/server/handlers/porter_app/run_app_job.go (hunk 1 above)
Comment (ianedwards) [reply api/server/handlers/porter_app/run_app_job.go:96]: don't have a preference; remember having used this helper previously

**Pull request merged**